### PR TITLE
fix(docs): fix link gitee sites confirm keep appearing

### PR DESCRIPTION
**Pull request by btea**
Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

When I  visit the site https://element-plus.org/， the confirm link to domestic site appear. After I click link, the confirm will appear. Such cycle...

### CODE REVIEW

File: docs/.vitepress/vitepress/components/vp-app.vue [hunk 1]
```diff
@@ -55,6 +55,7 @@ onMounted(async () => {
   )
 
   if (lang.value === 'zh-CN') {
+    if (location.href.startsWith('https://element-plus.gitee.io')) return
```
Comment (sxzz) [inline docs/.vitepress/vitepress/components/vp-app.vue:58]: Suggestion: we can use location.host to judge

Comment (btea) [reply docs/.vitepress/vitepress/components/vp-app.vue:58]: ok.

Comment (JeremyWuuuuu) [reply docs/.vitepress/vitepress/components/vp-app.vue:58]: Right, this was my big mistake sorry :bow:

**Review by jw-foss — APPROVED** [review]

**Comment by JeremyWuuuuu** [comment]
@btea @sxzz have left a comment, would you mind addressing it before we merge this?

**Review by sxzz — APPROVED** [review]

**Pull request merged**